client/src/components/ProjectSearchFilter.js

```diff
@@ -102,12 +102,22 @@ export const ProjectSearchFilter = ({
               />
             ))}
             {f === 'modalities' && (
-              <CheckBox
-                label={getReadable('has_bulk_rna_seq')}
-                value
-                checked={hasFilterOption('has_bulk_rna_seq')}
-                onChange={() => toggleFilterOption('has_bulk_rna_seq')}
-              />
+              <>
+                <CheckBox
+                  label={getReadable('has_spatial transcriptomics')}
```

Comment:
this can be removed from the filter, these will be auto generated by the API and the client in the above loop, this block is just to handle bulk rna seq because it is not treated the same as other modalities when we present it